bundles/org.openhab.binding.powermax/src/main/resources/OH-INF/thing/ip.xml

```diff
@@ -7,6 +7,7 @@
 	<bridge-type id="ip">
 		<label>IP Connection</label>
 		<description>This bridge represents the IP connection to the alarm system.</description>
+		<semantic-equipment-tag>NetworkAppliance</semantic-equipment-tag>
```

Comment:
It is not a network appliance, it is an alarm system, is there no tag appropriate for that ?